library/src/main/java/com/firebase/ui/com/firebasei/ui/authimpl/FirebaseOAuthToken.java

```diff
@@ -0,0 +1,29 @@
+package com.firebase.ui.com.firebasei.ui.authimpl;
+
+/**
+ * Created by abehaskins on 11/3/15.
+ */
+public class FirebaseOAuthToken {
+    public String token;
+    public String secret;
+    public String uid;
+    public String provider;
+    public int mode;
+
+    public static final int SIMPLE = 1;
+    public static final int COMPLEX = 2;
```

Comment:
I'm not sure about SIMPLE vs COMPLEX. Isn't it a OAuth v1 vs v2? Or simple vs composite?